### DISCUSSION TITLE
Changing nominal capacity does not affect real capacity

### DISCUSSION THREAD

**Discussion by kethan27** [pybamm-team/PyBaMM#3120] [Q&A] [answered]
I have updated nominal capacity of the parameter value of OKane2022. But while running the simulation the value is not getting updated. Can someone please help me out.
<img width="644" alt="image" src="https://github.com/pybamm-team/PyBaMM/assets/135948584/86461b29-c3ad-4b2f-b36a-f62e738fa08e">

**Comment by brosaplanella** [comment, marked as answer]
Nominal capacity is only used to convert current to C-rate, but does not affect the real capacity of the battery.

**Reply by kethan27** [reply]
Can we change the real capacity of battery?. In OKane2022 parameter set the capacity is given as 5.11Ah can I change it to 4.9Ah? If yes how to do it.

**Reply by brosaplanella** [reply]
You need to change other physical parameters that affect the capacity (e.g. electrode thickness, volume fraction...)
